packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.js

```diff
@@ -6,12 +6,12 @@ import _ from 'lodash';
 
 import {ITEM_TYPE_ACTIVITY, ITEM_TYPE_DAY_SEPARATOR, ITEM_TYPE_NEW_MESSAGE_SEPARATOR} from '@ciscospark/react-component-activity-list';
 
-const getConversationActivities = (state) => state.conversation.activities;
+const getConversationActivities = (state) => state.conversation.get(`activities`);
```

Comment:
yea, tried that out and it actually slowed things down because fires on every conversation change.